src/geometry/manifold/cont_mdiff_mfderiv.lean

```diff
@@ -94,6 +94,137 @@ lemma smooth.mdifferentiable_within_at (hf : smooth I I' f) :
   mdifferentiable_within_at I I' f s x :=
 hf.mdifferentiable_at.mdifferentiable_within_at
 
+/-! ### The derivative of a smooth function is smooth -/
+
+section mfderiv
+
+include Is I's Js
+
+/-- The function that sends `x` to the `y`-derivative of `f(x,y)` at `g(x)` is `C^n` at `x₀`,
+where the derivative is taken as a continuous linear map.
+We have to assume that `f` is `C^(n+1)` at `(x₀, g(x₀))` and `g` is `C^n` at `x₀`.
```

Comment:
The `n` does not correspond to the statement (where there is an `m` and an `n`). Can you adjust the docstring to match the statement? Same thing in the next lemma.